plugins/setting-resources/src/components/spaceTypes/editor/SpaceTypeGeneralSectionEditor.svelte

```diff
@@ -79,6 +81,17 @@
     }
     await ops.commit()
   }
+
+  async function changeMembers (members: Ref<Account>[]): Promise<void> {
```

Comment:
reduceCalls?

Reply:
Yeah, forgot that we have it already, will update the branch